src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -113,6 +113,12 @@ type ReportActionComposeProps = Pick<ComposerWithSuggestionsProps, 'reportID' |
 
     /** Whether the main composer was hidden */
     didHideComposerInput?: boolean;
+
+    /** The native ID for this component */
+    nativeID?: string;
+
+    /** Callback when layout of composer changes */
+    onLayout: (height: number) => void;
```

Comment:
Can we just accept the whole `onLayout` callback here? Then we don't need to create another `onLayoutInternal` wrapper here, and in `ReportFooter` we have to define the callback anyway